src/Jenssegers/Mongodb/Connection.php

```diff
@@ -93,7 +107,7 @@ public function table($table, $as = null)
      * @param string $name
      * @return Collection
      */
-    public function getCollection($name)
+    public function getCollection(string $name)
```

Comment:
I believe changes like this should be extracted to a separate PR to avoid side-effects.